.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -44,4 +44,10 @@ screenshots or screencasts related, please provide them here.
 If applicable, please attach the logs from running Mousai in the
 terminal with the following environment variables: `RUST_BACKTRACE=1 RUST_LOG=mousai=debug`
 (e.g., `RUST_BACKTRACE=1 RUST_LOG=mousai=debug flatpak run io.github.seadve.Mousai`).
```

Comment:
Make it also consistent with `Flatpak`, by instead of putting the flatpak instructions inside parentheses, put it also in its own paragraph.

Reply:
I think better to divide it in two paragraphs.. flatpak and snap